dataland-frontend/src/components/resources/frameworkDataSearch/PreviousReportsModal.vue

```diff
@@ -9,7 +9,7 @@
           class="row mb-2"
```

Comment:
maybe next time include the DALA number in the name of the PR :D

Reply:
right